auth/store_test.go

```diff
@@ -114,7 +114,7 @@ func setupAuthStore(t *testing.T) (store *authStore, teardownfunc func(t *testin
 		t.Fatal(err)
 	}
 
```

Comment:
Sorry for delay @mitake. Can we add some simple unit tests where `NoPassword == true`?

Reply:
@gyuho thanks for pointing, I'll add a new test case.